### DISCUSSION TITLE
How to Disable All Custom Nodes Except Selected Ones for a Specific Workflow in ComfyUI?

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [Comfy-Org/ComfyUI#4734] [Ideas]
Is there a way in ComfyUI to disable all custom nodes except for those I’m using in a specific workflow that I want to work on? I want to avoid using the --disable-all-custom-nodes command if it disables nodes that I still need.

**Comment by Amit30swgoh** [comment]
for example i dont want to load 100+ custom nodes only for segmention 
![image](https://github.com/user-attachments/assets/b1fe4fa1-bc37-4e8d-90e8-0c5a8f91ae15)
 especially while using google colab pro :)

**Comment by ltdrdata** [comment]
The active status of custom nodes is determined when ComfyUI starts, and it cannot be controlled on a per-workflow basis.

**Reply by Amit30swgoh** [reply]
> The active status of custom nodes is determined when ComfyUI starts, and it cannot be controlled on a per-workflow basis.

Thank you for the quick answer friend, keep up the great work Dr!
